lib/ansible/module_utils/service.py

```diff
@@ -149,6 +140,38 @@ def daemonize(module, cmd):
         if pid > 0:
             os._exit(0)
 
+    return pid
+
+
+def daemonize(module, cmd):
```

Comment:
Why are we doing all this extra boilerplate when multiprocessing.Process does it for us?

Reply:
this is mostly a copy of preexisting code, we should migrate to multiprocessing now that 2.6 is min target. But my original point is still valid, we should have 1 copy of this code.